src/lein_monolith/ansi.clj

```diff
@@ -0,0 +1,13 @@
+(ns lein-monolith.ansi
```

Comment:
Rather than adding a new namespace for this, I think this could go directly in `lein-monolith.task.util`.